<!DOCTYPE html>
<html><head><title>VerifyCloudException</title><link rel="stylesheet" href="../../styles.css"/><script src="../../scripts.js"></script></head><body onload="ro();">
<div class="rH">4 references to VerifyCloudException</div><div class="rA">OperationalInsights.Test (4)</div><div class="rG" id="OperationalInsights.Test"><div class="rF"><div class="rN">ScenarioTests\DataSourceOperationsTests.Scenario.cs (2)</div>
<a href="../ScenarioTests/DataSourceOperationsTests.Scenario.cs.html#96"><b>96</b>TestHelper.<i>VerifyCloudException</i>(HttpStatusCode.NotFound, () =&gt; client.DataSources.Get(resourceGroupName, workspaceName, dataSourceName));</a>
<a href="../ScenarioTests/DataSourceOperationsTests.Scenario.cs.html#174"><b>174</b>TestHelper.<i>VerifyCloudException</i>(HttpStatusCode.NotFound, () =&gt; client.DataSources.CreateOrUpdate(resourceGroupName, workspaceName, dataSourceName, createParameters));</a>
</div>
<div class="rF"><div class="rN">ScenarioTests\LinkedServiceOperationsTests.Scenario.cs (1)</div>
<a href="../ScenarioTests/LinkedServiceOperationsTests.Scenario.cs.html#97"><b>97</b>TestHelper.<i>VerifyCloudException</i>(HttpStatusCode.NotFound, () =&gt; client.LinkedServices.CreateOrUpdate(resourceGroupName, workspaceName, linkedServiceName, createParameters));</a>
</div>
<div class="rF"><div class="rN">ScenarioTests\WorkspaceOperationsTests.Scenario.cs (1)</div>
<a href="../ScenarioTests/WorkspaceOperationsTests.Scenario.cs.html#83"><b>83</b>TestHelper.<i>VerifyCloudException</i>(</a>
</div>
</div>
</body></html>